tempesta_fw/t/functional/helpers/control.py

```diff
@@ -198,6 +199,10 @@ def parse_out(self, stdout, stderr):
                               m.group(1))
             self.errors += (int(err_m.group(1)) + int(err_m.group(2))
                             + int(err_m.group(3)) + int(err_m.group(4)))
+            self.statuses["connect_error"] = int(err_m.group(1))
```

Comment:
Hard to say something on this change. Wrk-specific counters are exposed for use in generic code. It's not good, but workaround is required and a new tests structure is planned for the nearest future. So ok, but add a comment and link it with #996